extension/test/data_test.ts

```diff
@@ -69,4 +69,60 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    it('should return the text for a word dictionary entry', function () {
+      const entry = rcxDict.wordSearch('あ', /* doNames= */ false);
+      if (!entry) {
+        throw new Error(
+          `rcxDict.wordSearch('あ', /* doNames= */ false) returns null`
+        );
+      }
+      expect(rcxDict.makeText(entry, /* max= */ 1)).to.equal(
```

Comment:
```suggestion

      expect(rcxDict.makeText(entry, /* max= */ 1)).to.equal(
```

It's convenient to explicitly call out the Arrange Act Assert sections with a single new line between each part. (In this case Act and Assert are in the same statement which is fine imo since vertical space is precious)

https://github.com/goldbergyoni/javascript-testing-best-practices#-%EF%B8%8F-12-structure-tests-by-the-aaa-pattern

Reply:
Yeah, it might help with readability to have spaces in between the arrange part, the action part, and the assertion part. Thanks for the link. I'll try follow this style when writing tests.